schema.sql

```diff
@@ -0,0 +1,126 @@
+drop database if exists CareerLattice;
+create database CareerLattice;
+
+drop user if exists user@localhost;
+create user user@localhost identified with mysql_native_password by 'careerLattice!';
```

Comment:
Would be good to look into providing a hashed password here, so you don't have a plaintext password stored in GitHub.